README.md

```diff
@@ -158,15 +158,45 @@ this will generate a log at, for example,
 #### All Roots
 - Without profiling:
 ```ruby
-RAILS_ENV=production bundle exec rake c2m_check_version_all_dirs
+RAILS_ENV=production bundle exec rake c2m_check_version_all_dirs['2018-01-22 22:54:48 UTC']
 ```
 - With profiling:
 ```ruby
-RAILS_ENV=production bundle exec rake c2m_check_version_all_dirs[profile]
+RAILS_ENV=production bundle exec rake c2m_check_version_all_dirs['2018-01-22 22:54:48 UTC',profile]
 ```
 this will generate a log at, for example,
 ```log/profile_c2m_check_version_all_roots2018-01-01T14:25:31-flat.txt```
 
+### Run Checksum Validation for a single root or for all storage roots
+- Parse all manifestInventory.xml and most recent signatureCatalog.xml for stored checksums and verify against computed checksums.
+
+- To run rake tasks below, give a date, the name of the endpoint (e.g. from settings/development.yml) and an algorithm (md5, sha1, sha256) as arguments.
+
+- Note: Must enter date/timestamp argument as a string.
+
+#### Single Root
+- Without profiling
+```ruby
+RAILS_ENV=production bundle exec rake cv_single_endpoint['2018-01-22 22:54:48 UTC',fixture_sr1,'MD5']
```

Comment:
so fixture_sr1 should NOT have quotes, right?

Reply:
correct. I'm not 100% sure which Endpoint this will be referring to, but i assumed that it is the same as the C2M and M2C. I grabbed it from the `developement.yml`
```
moab:
  # this is a list of named storage roots.  the names are used to seed the endpoints table.
  storage_roots:
    # name: '/location/on/disk'  # e.g.
    fixture_sr1: 'spec/fixtures/storage_root01'
    fixture_sr2: 'spec/fixtures/storage_root02'
workflow_services_url: 'https://sul-lyberservices-test.stanford.edu/workflow/'
```